src/platform/macos/view.rs

```diff
@@ -38,8 +39,14 @@ pub fn new_view(window: id, shared: Weak<Shared>) -> IdRef {
     unsafe {
         // This is free'd in `dealloc`
         let state_ptr = Box::into_raw(Box::new(state)) as *mut c_void;
-        let view: id = msg_send![VIEW_CLASS.0, alloc];
-        IdRef::new(msg_send![view, initWithWinit:state_ptr])
+        let view_class = if win_attribs.blur { VISUAL_EFFECT_VIEW_CLASS.0 } else { VIEW_CLASS.0 };
+        let view: id = msg_send![view_class, alloc];
+        msg_send![view, initWithWinit:state_ptr];
+        if win_attribs.blur {
+            msg_send![view, setMaterial: BlurMaterial::Light as i64];
```

Comment:
You have to do `let _: () = msg_send!`, as otherwise segsegv/sigill will be triggered at runtime when the `!` (never) type is stabilized.